packages/dd-trace/src/proxy.js

```diff
@@ -104,6 +105,11 @@ class Tracer extends NoopProxy {
           this._profilerStarted = profiler.start(config)
         } catch (e) {
           log.error(e)
+          telemetryLog.publish({
+            message: e.message,
+            level: 'ERROR',
+            stack_trace: e.stack
+          })
```

Comment:
`telemetryLog.hasSubscribers` check?

```suggestion
          if (telemetryLog.hasSubscribers) {
            telemetryLog.publish({
              message: e.message,
              level: 'ERROR',
              stack_trace: e.stack
            })
          }
```

Reply:
I figured this happens at most once so the system can tolerate one no-op invocation in exchange for slightly lower code complexity.